packages/camera/camera/README.md

```diff
@@ -80,6 +80,28 @@ void didChangeAppLifecycleState(AppLifecycleState state) {
 }
 ```
 
+### Handling camera access permissions
+
+Permission errors may be thrown when initializing the camera controller, and you are expected to handle them properly.
+
+Here is a list of all permission error codes that can be thrown:
+
+- CameraAccessDenied
+Thrown when user denied the camera access permission.
+
+- CameraAccessDeniedWithoutPrompt
+iOS only for now.
+Thrown when user has previously denied the permission.
+iOS does not allow prompting alert dialog a second time. Users will have to go to Settings > Privacy in order to enable camera access.
+
+- CameraAccessRestricted
+iOS only for now.
+Thrown when camera access is restricted and users cannot grant permission (parental control).
+
+- cameraPermission
```

Comment:
This seems not consistent with other exceptions cases? Is this intentional?

Reply:
This is a legacy code for Android and Web that we will remove soon. I still include it because with this current version we will need to handle them.